iroha-cli/CMakeLists.txt

```diff
@@ -64,6 +64,9 @@ target_link_libraries(iroha-cli
     ${Boost_SYSTEM_LIBRARY}
     ${Boost_FILESYSTEM_LIBRARY}
     )
+target_include_directories(iroha-cli PUBLIC
+    ${Boost_INCLUDE_DIRS}
```

Comment:
The target for boos looks horrible. We should rework it soon.

Reply:
why? looks good to me